kitsune/search/es_utils.py

```diff
@@ -134,10 +159,15 @@ def index_object(doc_type_name, obj_id):
         # just return
         return
 
+    kwargs = {}
+    # For ES8, use string "true" instead of boolean True for refresh parameter
+    if settings.TEST:
+        kwargs["refresh"] = "true"
```

Comment:
What's the purpose of this argument?

Reply:
It forces ES to refresh, making changes immediately visible to search. So it helps for local testing more than anything else.